core/storage/user/gae_models.py

```diff
@@ -672,3 +673,85 @@ def construct_model_id(cls, user_id, skill_id):
             str. The model id corresponding to the given user and skill.
         """
         return '%s.%s' % (user_id, skill_id)
+
+
+class UserContributionScoringModel(base_models.BaseModel):
+    """Model for storing the scores of a user for various suggestions created by
+    the user. Users having scores above a particular threshold for a category
+    can review suggestions for that category.
+
+    The id for this model is of the form '{{score_category}}.{{user_id}}'.
+    """
+
+    # The user id of the user.
+    user_id = ndb.StringProperty(required=True, indexed=True)
+    # The category of suggestion to score the user on.
+    score_category = ndb.StringProperty(required=True, indexed=True)
```

Comment:
What are examples of possible score categories?

Reply:
Possible examples are 'content.Algebra', 'content.Architecture', 'content.<exploration_category>', 'translation.English', 'translation.French', 'translation.<language>', 'question.<topic_id>'. These are the ones for the use cases that we have thought of (which covers all of the use cases we can pinpoint at the moment). In case new situations pop up, new score categories will also be needed.